megamek/src/megamek/common/Mounted.java

```diff
@@ -1198,7 +1198,8 @@ public int getExplosionDamage() {
                     && curMode().equals("Powered Down")) {
                 return 0;
             }
-            if (isHotLoaded() && (getLinked().getUsableShotsLeft() > 0)) {
+            if ((isHotLoaded() || hasQuirk(OptionsConstants.QUIRK_WEAP_NEG_AMMO_FEED_PROBLEMS))
```

Comment:
Missile launchers now have explosion damage if the weapon has ammo feed problems.